aes/src/ni/aes192.rs

```diff
@@ -1,11 +1,11 @@
-use crate::arch::*;
+use crate::ni::arch::*;
 use cipher::{
     consts::{U16, U24, U8},
     generic_array::GenericArray,
     BlockCipher, NewBlockCipher,
 };
 
-use crate::utils::{Block128, Block128x8};
+use crate::ni::utils::{Block128, Block128x8};
```

Comment:
Alternatively we could use `super` instead of `crate::ni`. It's a bit more compact and will require less changes if we are to move or rename the module.

Reply:
Changed all the ones that seem relevant in 2ea345f